src/mongo/db/audit/audit.cpp

```diff
@@ -86,16 +96,19 @@ namespace audit {
         virtual void rotate() = 0;
     };
 
-    // Writes audit events to a json file
-    class JSONAuditLog : public WritableAuditLog {
+    // Writes audit events to a file
+    class FileAuditLog : public WritableAuditLog {
         bool ioErrorShouldRetry(int errcode) {
             return (errcode == EAGAIN ||
                     errcode == EWOULDBLOCK ||
                     errcode == EINTR);
         }
 
+    protected:
+        virtual AuditLogFormatAdapter *getAdapter(const BSONObj &obj) const = 0;
```

Comment:
Since this method creates `AuditLogFormatAdapter` and passes ownership on it, it's better to add the corresponding comment for implementors, and also rename the method to `createAdapter`.